src/conv2d/CMakeLists.txt

```diff
@@ -438,3 +438,9 @@ snn_object_library(
   SOURCES
     selector/default_selector.cc
 )
+
+if (WIN32)
+  # Need to define sycl_dnn_EXPORTS so the symbols in 
+  # selector/default_selector.cc are exported
+  target_compile_definitions(selector_conv2d PUBLIC sycl_dnn_EXPORTS)
```

Comment:
This should not be needed any more with the changes to the object library export.